Neo4j.Driver/Neo4j.Driver/Internal/IO/BoltReader.cs

```diff
@@ -81,14 +81,22 @@ public Task ReadAsync(IMessageResponseHandler responseHandler)
                 _chunkReader.ReadNextMessageAsync(_bufferStream)
                     .ContinueWith(t =>
                     {
-                        ProcessMessage(responseHandler);
+                        ConsumeMessages(responseHandler);
                     }, TaskContinuationOptions.ExecuteSynchronously);
         }
 
-        private void ProcessMessage(IMessageResponseHandler responseHandler)
+        private void ConsumeMessages(IMessageResponseHandler responseHandler)
         {
             _bufferStream.Position = 0;
 
+            while (_bufferStream.Length > _bufferStream.Position)
```

Comment:
I am a bit confused as I did not get why this will ever be false???
I thought _bufferStream (MemoryStream) is infinite, isn't it?